components/fieldlabel/index.css

```diff
@@ -59,50 +98,74 @@ governing permissions and limitations under the License.
   font-smoothing: subpixel-antialiased;
 }
 
+/* international lang support */
+.spectrum-FieldLabel {
+
+  &:lang(ja),
+  &:lang(zh),
+  &:lang(ko) {
+    line-height: var(--mod-fieldlabel-CJK-line-height, var(--spectrum-fieldlabel-cjk-line-height));
+  }
+}
+
 .spectrum-FieldLabel-requiredIcon {
-  margin-block: 0; /* Previously used this improperly, it doesn't reposition asterisk, just increased top margin on the whole field label var(--spectrum-fieldlabel-asterisk-margin-y) */
-  margin-inline: var(--spectrum-fieldlabel-asterisk-gap) 0;
+  margin-block: 0;
+  /* Previously used this improperly, it doesn't reposition asterisk, just increased top margin on the whole field label var(--spectrum-fieldlabel-asterisk-margin-y) */
+  margin-inline: var(--mod-fieldlabel-asterisk-gap, var(--spectrum-fieldlabel-asterisk-gap)) 0;
 }
 
 .spectrum-FieldLabel--left {
   display: inline-block;
-  padding-block: var(--spectrum-fieldlabel-m-side-padding-top) 0;
-  padding-inline: 0 var(--spectrum-fieldlabel-m-side-padding-right);
+  padding-block: var(--mod-fieldlabel-side-padding-top, var(--spectrum-fieldlabel-side-padding-top)) 0;
```

Comment:
This seems to be a problem for Web components, the padding might be too small. 
![2022-09-22_14-50-55 (1)](https://user-images.githubusercontent.com/52184321/192054693-ba6ba429-3449-4a20-b253-5423ce561ff7.gif)